### ISSUE TITLE
Failed to read text from Accessibility.txt

### ISSUE THREAD

**Issue by tilucasoli** [JMateusSousa/CBL-Guide#2]
## Autores
@jessicaguiot e @tilucasoli

## Proposta de alteração
Para que os arquivos responsáveis por armazenar os textos utilizados sejam lidos em conjunto com o script, sem a necessidade de armazena-los em uma pasta separada do executável, sugerimos que seja criado um arquivo `.bundle` que terá a função de guardar dados essenciais para a execução do programa.

## Motivo da proposta
Durante a execução do script é apresentado o error: **Failed to read text from Accessibility.txt** que compromete a usabilidade do programa.

## Como fazer

Para a criação do `ResourcesBundle.bundle`:
1. `File`>`New`>`Target`>`Bundle` e daremos o nome de ResourcesBundle 

2. Após criar o Bundle iremos arrastar a pasta Files para dentro da pasta ResourcesBundle.
<img width="215" alt="image" src="https://user-images.githubusercontent.com/62367544/78391368-9c9c1e80-75bc-11ea-9fd0-12b12be0f978.png">

3. Vocês devem selecionar todos os arquivos `.txt` dentro de `File`, e na aba direita selecionar a caixinha de **ResourcesBundle**
<img width="260" alt="Screen Shot 2020-04-03 at 15 10 57" src="https://user-images.githubusercontent.com/62367544/78391734-54313080-75bd-11ea-8d3a-16045144b0ba.png">

4. Clicar em `CBLGuide.xcodeproj` representado pelo ícone azul acima das pastas do projeto

5. Na categoria Targets escolham o executável `CBLGuide` > `Build Phases` > `Copy Files` > adicionaram o arquivo `ResourcesBundle.bundle` 
<img width="1280" alt="Screen Shot 2020-04-03 at 15 21 36" src="https://user-images.githubusercontent.com/62367544/78392577-d2da9d80-75be-11ea-8877-0ea71781ba76.png">

**Parabéns 🥳, seu bundle foi criado**

### Scripts Adicionais
Para finalizar deve-se ler os arquivo dentro do bundle, devido a isso crie a seguinte função:
```swift
func getRootURL() -> URL {
        let currentDirectory = URL(fileURLWithPath: FileManager.default.currentDirectoryPath)
        let bundleURL = URL(fileURLWithPath: "ResourcesBundle.bundle", relativeTo: currentDirectory)
        let bundle = Bundle(url: bundleURL)!
        let rootPath = bundle.url(forResource: nil, withExtension: "txt", subdirectory: nil)
        return rootPath!.baseURL!
    }
```

Para adequar o código ao novo método tem-se que acrescentar na primeira linha da função `access()` o trecho de código a seguir:

```swift
let url = self.getRootURL()
```

**Comment by JMateusSousa** [comment]
Pronto, sua sugestão foi bem recebida e foi implementada.
Logo essa versão terá seu commit.
